packages/utils/src/phoneNumbers.ts

```diff
@@ -56,9 +59,13 @@ export function getCountryCode(e164PhoneNumber: string) {
     return null
   }
   try {
-    return phoneUtil.parse(e164PhoneNumber).getCountryCode()
+    // type CountryCallingCode is an extension of String, but
+    // type checks fail when attempting to pass it as a parameter for parseInt,
+    // so here we force the string representation, and then parse it to a number
+    const callingCode = parsePhoneNumberFromString(e164PhoneNumber)?.countryCallingCode
+    return parseInt(callingCode as string, 10)
   } catch (error) {
-    console.debug(`getCountryCode, number: ${e164PhoneNumber}, error: ${error}`)
+    console.debug(`countryCallingCode, number: ${e164PhoneNumber}, error: ${error}`)
```

Comment:
Why change this? We use this convention of starting with the method name (or tag and method name) very often.